onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -129,8 +130,12 @@ namespace perftest {
       "\t    [NNAPI only] [NNAPI_FLAG_CPU_ONLY]: Using CPU only in NNAPI EP.\n"
       "\t    [Example] [For NNAPI EP] -e nnapi -i \"NNAPI_FLAG_USE_FP16 NNAPI_FLAG_USE_NCHW NNAPI_FLAG_CPU_DISABLED\"\n"
       "\n"
-      "\t    [CoreML only] [COREML_FLAG_CREATE_MLPROGRAM COREML_FLAG_USE_CPU_ONLY COREML_FLAG_USE_CPU_AND_GPU]: Create an ML Program model instead of Neural Network.\n"
-      "\t    [Example] [For CoreML EP] -e coreml -i \"COREML_FLAG_CREATE_MLPROGRAM\"\n"
+      "\t    [CoreML only] [ModelFormat]:[MLProgram, NeuralNetwork] Create an ML Program model or Neural Network.\n"
+      "\t    [CoreML only] [ComputeUnits]:[MLComputeUnitsCPUAndNeuralEngine MLComputeUnitsCPUAndGPU MLComputeUnitsCPUAndGPU MLComputeUnitsCPUOnly] the backend device to run model.\n"
```

Comment:
If the key name is 'MLComputeUnits', the prefix on all the values should be redundant and a user won't have to type as much.

```suggestion
      "\t    [CoreML only] [MLComputeUnits]:[CPUAndNeuralEngine CPUAndGPU CPUAndGPU CPUOnly] Specify to limit the backend device/s used to run the model.\n"
```